litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml

```diff
@@ -165,6 +165,8 @@ spec:
               value: "litmuschaos/chaos-operator:1.13.0"
             - name: LITMUS_CHAOS_RUNNER_IMAGE
               value: "litmuschaos/chaos-runner:1.13.0"
+            - name: LITMUS-CHAOS-EXPORTER_IMAGE
+              value: "litmuschaos/chaos-exporter:v1.13.x"
```

Comment:
Is it the image tag- v1.13.x

Reply:
yes @rajdas98

Reply:
I don't think we create this kind of image tag.

Reply:
I have been using this, here is another PR for the same, https://github.com/litmuschaos/litmus/pull/2438 cc: @ispeakc0de @ksatchit

Reply:
That does not work, i have tried using that tag @rajdas98

Reply:
1.13.0 is present in dockerhub. https://hub.docker.com/r/litmuschaos/chaos-exporter/tags?page=1&ordering=last_updated

Reply:
okay i am updating it, i tried `v1.13.0`